soh/soh/Enhancements/debugger/debugSaveEditor.h

```diff
@@ -1,3 +1,414 @@
 #pragma once
 
+#include <array>
+#include <map>
+#include <string>
+#include <vector>
+#include <soh/Enhancements/randomizer/randomizer_inf.h>
+
 void InitSaveEditor();
+
+typedef enum {
+    EVENT_CHECK_INF,
+    ITEM_GET_INF,
+    INF_TABLE,
+    EVENT_INF,
+    RANDOMIZER_INF,
+} FlagTableType;
+
+typedef struct {
+    const char* name;
+    FlagTableType flagTableType;
+    uint32_t size;
+    std::map<uint16_t, const char*> flagDescriptions;
+} FlagTable;
+
+// Reference https://tcrf.net/Proto:The_Legend_of_Zelda:_Ocarina_of_Time_Master_Quest/Event_Editor
```

Comment:
TCRF is based on leaks. Not sure we can even have this.
Thoughts? @NEstelami @Rozelette @louist103 @dcvz @briaguya-ai

Reply:
```suggestion
// Reference https://tcrf.net/Proto:The_Legend_of_Zelda:_Ocarina_of_Time_Master_Quest/Event_Editor
// This information predates gigaleak and is considered safe to use
```